### PR TITLE
Modified API Creation forms

### PR DESCRIPTION
**Pull request by HiranyaKavishani**
Fixed https://github.com/wso2/product-apim/issues/6283

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx [hunk 1]
```diff
@@ -254,6 +244,16 @@ function APICreateDefault(props) {
...
                             </Link>
                         </Grid>
+                        <Grid item>
+                            <Button
+                                variant='contained'
+                                color='primary'
+                                disabled={!apiInputs.isFormValid || isCreating}
+                                onClick={createAPI}
+                            >
+                                Create {isCreating && <CircularProgress size={24} />}
```
Comment (rmsamitha) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx]: Shall we add permission checking condition to this button and other places too? For this button check how this is done in general API create button.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/OpenAPI/ApiCreateOpenAPI.jsx [hunk 2]
```diff
@@ -240,19 +257,6 @@ export default function ApiCreateOpenAPI(props) {
                                 </Button>
```
Comment (rmsamitha) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/OpenAPI/ApiCreateOpenAPI.jsx]: Shall we add permission checking condition to this button and other places too?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx (hunk 1 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx]: We won't add this to here as the logic comes with isCreating prop

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/OpenAPI/ApiCreateOpenAPI.jsx (hunk 2 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/OpenAPI/ApiCreateOpenAPI.jsx]: We won't add this to here as the logic comes with isCreating prop

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx (hunk 1 above)
Comment (rmsamitha) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx]: isCreating call is not doing the required permission check. Anyway that issue needs to address separately.

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx]: Yes. It has been addressed by the root level by disabling the "Create API" button.

**Pull request merged**